js/script.js

```diff
@@ -2,7 +2,7 @@ var script = {
   popcorn: null,
   init: function () {
 
-    var pop = Popcorn.smart("#videoClip", ["https://storage.cloud.google.com/hello-p5-bucket/p5-video-draft-3.webm", "https://storage.cloud.google.com/hello-p5-bucket/p5-video-draft-3.mp4"], { width: 1920, height: 1080 });
+    var pop = Popcorn.smart("#videoClip", ["/assets/p5-video-draft-4.webm", "/assets/p5-video-draft-4.mp4"], { width: 1920, height: 1080 });
```

Comment:
@tuantinghuang The public link is here: https://hello-assets.p5js.org/p5-video-draft-3.webm - please check if this works?

Reply:
And mp4: https://hello-assets.p5js.org/p5-video-draft-x264.mp4

Reply:
Both link works, thank you! I will send the newest version of the video files once I fix more things!